src/desktop/apps/article/components/App.js

```diff
@@ -40,7 +40,12 @@ export default hot(module)(
           )
         }
         default: {
-          return <ArticleLayout {...this.props} />
+          return (
+            <ArticleLayout
+              seriesArticle={article.seriesArticle}
```

Comment:
Is this necessary anymore? I'm following the props in Reaction for `seriesArticle` and it doesn't appear to be used directly. (Only indirectly through props.article)